react/src/views/BoxEdit/BoxEditView.tsx

```diff
@@ -185,19 +184,15 @@ function BoxEditView() {
     }));
 
   if (allLocations == null) {
-    notificationVar({
-      title: "Error",
-      type: "error",
-      message: "Error: No other locations are visible!",
+    triggerError({
+      message: "No locations are available!",
     });
     return <div />;
   }
 
   if (productAndSizesData == null) {
-    notificationVar({
-      title: "Error",
-      type: "error",
-      message: "Error: No products are visible!",
+    triggerError({
+      message: "No products are available!",
```

Comment:
I tested this message by setting `const productAndSizesData = null;` above; I then see the message three times, not sure if that's due to recent changes or if that's been a bug before.

Reply:
Also in case of an error in BoxEditView or BoxEdit, can we return to BoxView of that box?

Reply:
Clarified: @HaGuesto integrates `useEffect` to avoid multiple notifications displayed.

Reply:
I added the useEffect and error messages only appear once on my side.
Regarding the forwarding I did not find a quick solution and left it for now.

Reply:
Also works on my end :+1: